developer/src/kmc-generate/test/fixtures/lexical-model/sample.en.sample/source/readme.htm

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Sample Project</title>
+  <style type="text/css">
+    p { font: 10pt Tahoma; }
+    h1 { font: bold 16pt Tahoma; color: #4444cc; margin-bottom: 2px }
+    h2 { font: bold 12pt Tahoma; color: #4444cc; }
+  </style>
+</head>
+<body>
+
+<h1>Sample Project</h1>
+
+<p>
+  # A mighty description
```

Comment:
1. Is the `#` intentional and generated?
2. Could a generated description be something like "<name> model for <language name>"?

Reply:
1. Yes, the '#' is part of the command line passed in: https://github.com/keymanapp/keyman/blob/4c0588ea393f0fecdb1a6fa642956922f7865a50/developer/src/kmc-generate/test/test-lexical-model-generator.ts#L18
2. It doesn't add enough to be worthwhile -- if description is omitted, just uses the name of the model.